app/src/app/components/login/Login.tsx

```diff
@@ -1,8 +1,14 @@
 import { UserAuthForm } from "./UserAuthForm";
+import { useSearchParams } from "react-router-dom";
 
 export default function Login() {
+  const [searchParams] = useSearchParams();
+  const sessionExpiryMsg = searchParams.get("info");
```

Comment:
```suggestion
  const sessionExpiryMessage = searchParams.get("expiryMessage");
```

Reply:
I'm using the more generic search param 'info' by analogy to how elsewhere we're using the search param 'success' or 'error'. I'm thinking of alerts/notices as coming in three or four types - success (green), error (red), info (blue) and warning (yellow); we currently only use the first two.

Reply:
cool if that's the case should the variable be renamed? like `infoMessage`  or something

Reply:
My model is
```ts
  const updatePasswordSuccess = searchParams.get("success");
```
in BasicUserAuthForm.tsx

So I'll rename the variable so it mentions 'info', but I think it makes sense to be specific about what kind of message we're expecting in this template (a message about session expiry)